src/GraphQL/AssetType/AssetFolderType.php

```diff
@@ -64,7 +64,19 @@ public function build(&$config)
                     ]
                 ],
                 'resolve' => [$elementResolver, "resolveProperties"]
-            ]
+            ],
+            'parent' => [
+                'type' => $this,
+                'resolve' => [$elementResolver, "resolveParent"],
+            ],
+            'children' => [
+                'type' => Type::listOf($this),
+                'resolve' => [$elementResolver, "resolveChildren"],
+            ],
+            'siblings' => [
```

Comment:
siblings is not a reserved word so it can clash with a pimcore class attribute

Reply:
what do you suggest? adding sibling(s) to the reserved words list (which would be a breaking change and would have to be done in [the core](https://github.com/pimcore/pimcore/)) or use another key (seems like none of the reserved words fits for this) or discard the functionality of getting siblings via data-hub for now?

Reply:
I think you could prepend it with an underscore (which would also make it clear that it is not part of the data model :-)